elastic/kibana
# [Fleet] Force unenroll in Agent activity

## Summary

Fix https://github.com/elastic/kibana/issues/140794 and https://github.com/elastic/kibana/issues/140923

- Added an action document with action results on force unenroll, so that Agent activity shows that force unenroll was successful.
- On force unenroll, checking other non-expired unenroll actions and adding action results for agents that were force unenrolled, so that the unenroll action is not stuck in progress.

To verify:
- unenroll a few agents with fleet server stopped
- check in agent activity that there is an in progress unenroll activity
- force unenroll the same agents
- check in agent activity that the previous unenroll action shows up as completed
- check that there is a completed force unenroll activity as well

<img width="1214" alt="image" src="https://user-images.githubusercontent.com/90178898/191493916-9e2db10e-6e95-484e-8714-31dd23c011ad.png">
<img width="643" alt="image" src="https://user-images.githubusercontent.com/90178898/191493999-7c8706c5-a855-4c56-a518-5c0455604411.png">



### Checklist
